src/Rule/Callback.php

```diff
@@ -39,4 +43,11 @@ protected function validateValue($value, ValidationContext $context = null): Res
         }
         return $result;
     }
+
+    public function withInjector(Injector $injector): self
```

Comment:
Is there a way to avoid manually passing injector? It's not too bad. Just asking.

Reply:
I did some research I got that we can have one more variant when Injector will be passed to Rule through Validator. 
It looks stranger than manual passing.